### ISSUE TITLE
git clone in the readme should use https:// URL

### ISSUE THREAD

**Issue by timhunt** [moodlehq/moodle-db-oracle#11]
I coudl not just copy-paste because I don't have write access to this repo.

**Comment by timhunt** [comment]
Also, for me

```
docker run --name db0 -p 1521:1521 moodlehq/moodle-db-oracle-r2
```

is failing, with

```
$ docker run --name db0 -p 1521:1521 moodlehq/moodle-db-oracle-r2
docker: Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Post http://%2Fvar%2Frun%2Fdocker.sock/v1.24/containers/create?name=db0: dial unix /var/run/docker.sock: connect: permission denied.
See 'docker run --help'.
```

**Comment by stronk7** [comment]
Hi @timhunt,

thanks for the report!

I've just merged #12 that fixes the URL. Also, I've tried here the image, ensuring that I'm using the repository one... and it's working without problems here. That error looks like something wrong in your local files via socket?

![oracle-working-ok](https://user-images.githubusercontent.com/167147/164215772-ac2f5a59-d633-4117-a8ec-a8bfb41c64bc.png)

I think I'm going to close this now. If there is anything else we can try or do, let me know.

Ciao :-)
